t/43-cli-schedule.t

```diff
@@ -0,0 +1,83 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Test::Most;
+use Test::Warnings qw(:report_warnings warning);
+use Mojo::Base -strict, -signatures;
+
+use FindBin;
+use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
+
+use OpenQA::Test::TimeLimit '7';
+use OpenQA::CLI;
+use OpenQA::CLI::schedule;
+use OpenQA::Jobs::Constants;
+use OpenQA::Test::Case;
+use Mojo::Server::Daemon;
+use Mojo::File qw(tempdir tempfile);
+use Test::Output qw(combined_like);
+use Test::MockModule;
+
+my ($archive, $res) = OpenQA::CLI::schedule->new;
```

Comment:
Is this a typo? A call to `->new` should never return more than one result. 🤔

Reply:
And the name `$archive` is inherited from the other command? Probably wanto to name it `$schedule` here.

Reply:
It managed to confuse me. 😁